src/internal/HtmlColor.h

```diff
@@ -76,6 +90,23 @@ struct HtmlColor
     };
 
     // ------------------------------------------------------------------------
+    // Parse a HTML4/CSS3 color name
+    //
+    // name - the color name
+    //
+    // It accepts all standard HTML4 names and, if USE_CSS3_COLORS macro is
+    // defined, the color names defined in CSS3 also
+    // ------------------------------------------------------------------------
+    bool Parse(const char* name);
```

Comment:
Do you think it will be useful? I'm using this function to parse strings supplied by Apple Siri; for constants string, you could just use the RGB constructor. Another option would be to export all colors defined in HtmlColorNames.cpp as constant HtmlColor objects and declare them in the include file.